### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by daPhipz**
Since I am half German and half French, I was happy to see this extension now has a French translation as well :)

I looked over the translation and found some minor things that might need changing before the next release (See the comments in the file).
For some reason, I can't ping Tuxman2 - I'll just comment on the original issue and hope that they'll get the notification ;)

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -39,7 +41,7 @@ msgstr "Réaffichage non corrigé"
 
 #: extension.js:106
 msgid "Reordering folder contents"
-msgstr "Réorganisation du contenu du dossier"
+msgstr "Réorganisation du contenu des dossiers"
```
Comment (daPhipz) [inline po/fr.po:44]: This is about the folders on the app grid, that users can create. Since there can be more than one, I'd use the plural here.

**Comment by stuarthayhurst** [comment]
Thanks for catching this!
By any chance you didn't notice anything about pull request templates when filing this one did you? I'm trying to add some templates to the repository but the PR template isn't showing up :/

File: po/fr.po [hunk 2]
```diff
@@ -135,7 +137,7 @@ msgstr "Alphabétique"
 #: ui/prefs-gtk4.ui:15 ui/prefs.ui:18
 msgctxt "folder-position"
 msgid "Start"
-msgstr "Démarrer"
+msgstr "Début"
```
Comment (daPhipz) [inline po/fr.po:140]: This is a value of the setting whether folders should be located at the start or end of the app grid, or in alphabetical order mixed with other applications. So I think "Début" is the word to use here.
Does "fin" need to be changed then as well, or is this OK (my French is a bit rusty, sadly...)?

**Review by daPhipz — COMMENTED** [review]
These are my proposed changes/questions.
Tuxman2, it'd be awesome if you could give feedback on this - thanks!

**Comment by daPhipz** [comment]
> By any chance you didn't notice anything about pull request templates when filing this one

I didn't, but maybe I was just not looking hard enough ;)
But I think that PR templates are a bit tricky to work with - AFAIK, you need to append some weird strings to the URL in order to make them work... But this info might be outdated. I'll do a nonsense PR in a couple of minutes and check if it works!

**Comment by stuarthayhurst** [comment]
> > By any chance you didn't notice anything about pull request templates when filing this one
> 
> I didn't, but maybe I was just not looking hard enough ;)
> But I think that PR templates are a bit tricky to work with - AFAIK, you need to append some weird strings to the URL in order to make them work... But this info might be outdated. I'll do a nonsense PR in a couple of minutes and check if it works!

I got them working :) GitHub's documentation seems to be off, the filename and location aren't quite right in their docs. Seems to be known about, just took a second of digging though.

**Comment by daPhipz** [comment]
Another thing that I saw:

```
#: prefs.js:119
msgid "About"
msgstr "À propos de"
```

I think this string could be translated another way. It is the label of a button that opens the About dialog with all the copyright info, links to this repository etc.
@Tuxman2, when you open `gnome-terminal` - what does it say instead of "Info" in the menu pictured below here?

Bildschirmfoto vom 2021-08-29 17-32-40

**Comment by stuarthayhurst** [comment]
This will probably need updating after #36 is merged, won't it?

**Comment by daPhipz** [comment]
Yes, it will - I'll take care of that ;)

**Comment by daPhipz** [comment]
Merged current master branch after #36 got accepted

File: po/fr.po (hunk 1 above)
Comment (afranke) [reply po/fr.po:48]: Oui.

File: po/fr.po (hunk 2 above)
Comment (afranke) [reply po/fr.po]: Oui, « Début » is correct here. I don’t understand what you want to change « Fin » to. « Fin » is the correct opposite of « Début » and translation for “End”.

**Comment by afranke** [comment]
"About" should be "À propos".

File: po/fr.po (hunk 1 above)
Comment (daPhipz) [reply po/fr.po:48]: Merci ;)

File: po/fr.po (hunk 2 above)
Comment (daPhipz) [reply po/fr.po]: Merci ;)

**Comment by daPhipz** [comment]
I updated the translation with help from @afranke over at the GNOME l10n IRC, it is ready now :tada:

**Comment by stuarthayhurst** [comment]
Thanks for doing this :D

**Pull request merged**